packages/nextjs/components/example-ui/ContractData.tsx

```diff
@@ -38,19 +37,19 @@ export const ContractData = () => {
     },
   });
 
-  const {
-    data: myGreetingChangeEvents,
-    isLoading: isLoadingEvents,
-    error: errorReadingEvents,
-  } = useScaffoldEventHistory({
-    contractName: "YourContract",
-    eventName: "GreetingChange",
-    fromBlock: Number(process.env.NEXT_PUBLIC_DEPLOY_BLOCK) || 0,
-    filters: { greetingSetter: address },
-    blockData: true,
-  });
+  // const {
```

Comment:
I think its commented due to infinite render issue, just adding a comment so that we don't forgot to uncomment before merging 🙌

Reply:
Since you back-merged main here, this should be fixed now